flutter/test/default_integrations_test.dart

```diff
@@ -72,7 +73,61 @@ void main() {
 
     verify(await fixture.hub.captureEvent(captureAny));
 
-    expect(true, called);
+    expect(called, true);
+  });
+
+  test('FlutterErrorIntegration captureEvent only called once', () async {
+    var numberOfDefaultCalls = 0;
+    final defaultError = (FlutterErrorDetails errorDetails) async {
+      numberOfDefaultCalls++;
+    };
+    FlutterError.onError = defaultError;
+
+    when(fixture.hub.captureEvent(captureAny))
+        .thenAnswer((_) => Future.value(SentryId.empty()));
+
+    final details = FlutterErrorDetails(exception: StateError('error'));
+
+    final integrationA = FlutterErrorIntegration();
+    integrationA.call(fixture.hub, fixture.options);
+    integrationA.close();
+
+    final integrationB = FlutterErrorIntegration();
+    integrationB.call(fixture.hub, fixture.options);
+
+    FlutterError.reportError(details);
+
+    verify(await fixture.hub.captureEvent(captureAny)).called(1);
+
+    expect(numberOfDefaultCalls, 1);
+  });
+
+  test('FlutterErrorIntegration close restored default onError', () {
+    final defaultOnError = (FlutterErrorDetails errorDetails) async {};
+    FlutterError.onError = defaultOnError;
+
+    final integration = FlutterErrorIntegration();
+    integration.call(fixture.hub, fixture.options);
+    expect(false, defaultOnError == FlutterError.onError);
+
+    integration.close();
+    expect(defaultOnError == FlutterError.onError, true);
```

Comment:
shouldn't we use the equal check only?

`expect(defaultOnError, FlutterError.onError);`